SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2383,6 +2418,27 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 			if (HandoverManager.IsValid())
 			{
 				HandoverManager->Advance();
+				for (Worker_EntityId Partition : HandoverManager->ConsumeDelegationLost())
```

Comment:
What's the advantage of having the reader consume the list rather than having it clear on the next advance call in the same way the view delta does?

Reply:
Or more generally if we want this kind of manipulation of the internal state of the delegation stuff, why not have this encapsulated in the same manager the owns the state and have an external api to hook into delegation events?

Reply:
1) You do not loose events. Making several advance calls won't stomp events that have not been processed.
2) I'm not sure I get the question.